packages/hw-transport-node-hid-noevents/src/TransportNodeHid.ts

```diff
@@ -15,27 +13,26 @@ import type { DeviceModel } from "@ledgerhq/devices";
 import { TransportError, DisconnectedDevice } from "@ledgerhq/errors";
 
 const filterInterface = (device) =>
-  ["win32", "darwin"].includes(process.platform)
-    ? // $FlowFixMe
-      device.usagePage === 0xffa0
+  ["win32", "darwin"].includes(process.platform) // $FlowFixMe
+    ? device.usagePage === 0xffa0
     : device.interface === 0;
 
-export function getDevices(): Array<*> {
+export function getDevices(): Array<any> {
```

Comment:
We should try to avoid using `any` at all costs, should we not return some kind of `Device` type here? I thought this was `Device` of [`@types/node-hid`](https://www.npmjs.com/package/@types/node-hid)? 

```typescript
export interface Device {
    vendorId: number;
    productId: number;
    path?: string;
    serialNumber?: string;
    manufacturer?: string;
    product?: string;
    release: number;
    interface: number;
    usagePage?: number;
    usage?: number;
}
```

Or some similar type?